### PR TITLE
[CU-jd850p] Add connection with slack bot

### PR DESCRIPTION
**Pull request by Zummek**

### CODE REVIEW

**Comment by Zummek** [comment]
Task linked: [CU-jd850p Make connection with slack](https://app.clickup.com/t/jd850p)

File: server/concepts/Slack.ts [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import { WebClient } from '@slack/web-api';
+
+const token = process.env.SLACK_APP_TOKEN;
+
+const slackConnection = new WebClient(token);
```
Comment (eastemployeer) [inline server/concepts/Slack.ts]: Shouldn't `slackConnection` be a property of class Slack?

Comment (Zummek) [reply server/concepts/Slack.ts]: I thought about it. I'm not sure. this is a static class hmm Hmm okej `Slack.connection` sound good

Comment (Zummek) [reply server/concepts/Slack.ts]: @eastemployeer please make review after changes

**Comment by eastemployeer** [comment]
Fine

**Review by eastemployeer — APPROVED** [review]

**Pull request merged**